web/star.dart

```diff
@@ -53,21 +51,27 @@ class Star implements Renderable {
       0.0
     ];
     gl.bufferData(
-        WebGL.ARRAY_BUFFER, new Float32List.fromList(vertices), WebGL.STATIC_DRAW);
+      WebGL.ARRAY_BUFFER,
+      new Float32List.fromList(vertices),
+      WebGL.STATIC_DRAW,
+    );
 
     starVertexTextureCoordBuffer = gl.createBuffer();
     gl.bindBuffer(WebGL.ARRAY_BUFFER, starVertexTextureCoordBuffer);
     var textureCoords = [0.0, 0.0, 1.0, 0.0, 0.0, 1.0, 1.0, 1.0];
     gl.bufferData(
-        WebGL.ARRAY_BUFFER, new Float32List.fromList(textureCoords), WebGL.STATIC_DRAW);
+      WebGL.ARRAY_BUFFER,
+      new Float32List.fromList(textureCoords),
+      WebGL.STATIC_DRAW,
+    );
   }
 
   void draw(
       {int vertex,
       int normal,
       int coord,
       UniformLocation color,
-      bool twinkle: false,
+      bool twinkle = false,
```

Comment:
Idle commentary / no requirements: I get why the dart team wanted to do this, but man I hate it... at least the call sights still look like a key/value map :)

Reply:
Aye. I'm curious on the back story on that one, but another time, another place.